src/client/BrowseLobbyModal.ts

```
@@ -0,0 +1,543 @@
+
+      const response = await apiFetch(url, {
+        method: "GET",
+        headers: { "Content-Type": "application/json" },
+      });
+
+      if (!response.ok) {
+        console.log(`Could not get game info for lobby ${lobbyId}: ${response.statusText}`);
+        return false; // If we can't get game info, assume it hasn't started
+      }
+
+      const gameInfo = await response.json() as { turns?: any[]; [key: string]: any };
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)